product-development/README.md

```diff
@@ -155,17 +141,29 @@ There may be rare cases where someone gets assigned to an epic that they
 don't want to work on, but in those cases we should explicitly call out
 that this is happening and why.
 
-At all times, every Tilter should have 1-2 epics assigned to them that haven't
-been started yet. The epics are their "Next Up" list.  This isn't final - they
-may get reassigned. But we bias towards keeping the current assignment so that
-Tilters know what's coming next.
+### DRIs 
 
-### Starting an Epic
+Each epic's assigned team has a [DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4), 
+set by Nick.
 
-When you're ready to start an epic, it should be clear which epic to start work
-on -- it's the highest-priority backlog epic assigned to you.
+The DRI is responsible for:
+- maintaining the epic state, description, and attached stories updated as the single source of truth of overall work status
+- reducing scope if additional work is discovered
+- driving the epic to completion
+
+The acceptance criteria sections are _especially critical_, including presenting
+the problem and final implementation in the [weekly epics
+meeting](#weekly-epics-meeting).
+
+### Specify Epic Team Members
 
-Move it into the `In Progress` state.
+Each epic has team members. This is specified in the description of the epic,
```

Comment:
Does the epic description reflect "current state" or "historical state"?

for example, when the epic is closed, should we remove all team members from the epic?

Worried that this would cause confusion. Maybe the calendar should be the source of truth instead, because it has a time component

Reply:
@nicks : For time component, I was thinking of using epic start date and epic end date as the source of truth of when it is actively being worked on. For team membership, what do you think about making all team members of an epic to be owners of the epic. And we only care about team membership for in progress epics? This would avoid the problem of having to remove team members from epics. (Once an epic is closed, we just don't look at membership.)

That way, if you just look at the in progress epics in Clubhouse, you can see, at a glance, which persons are part of which epic (team). Currently, I only typically make the DRI an owner of the epic. But Clubhouse epics can have multiple owners. And we can just annotate in the epic description who is the DRI. 

<img width="291" alt="Screen Shot 2021-01-19 at 6 40 58 PM" src="https://user-images.githubusercontent.com/10860550/105107315-e25fa000-5a85-11eb-9835-6f11a28d3e15.png">

Reply:
@nicks also there's a Gantt chart view for epics in Clubhouse. So I think it's easier to keep everything in Clubhouse as a source of truth, rather than messing around with Google calendar meetings. 

If that makes sense to you, I can probably whip up something with Clubhouse epic webhooks and Gcal API, to sync epic start dates and epic end dates into Gcal.